features/steps/shell.py

```diff
@@ -64,7 +64,17 @@ def when_i_run_command(
 ):
     command = process_template_vars(context, command)
     prefix = get_command_prefix_for_user_spec(user_spec)
-    full_cmd = prefix + shlex.split(command)
+    if "pro" in command.split():
+        command = command.replace(
+            "pro",
+            "python3 -m coverage run \
+                --source=/usr/lib/python3/dist-packages/uaclient /usr/bin/pro",
```

Comment:
I'd rather use `# noqa: E501` at the end of the line than break up a string like this to keep it less than 80 characters wide.